packages/terra-embedded-content-consumer/src/terra-dev-site/test/embedded-content-consumer/providers/DataStatusProvider.test.jsx

```diff
@@ -1,39 +0,0 @@
-/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Renamed to src/terra-dev-site/provider/embedded-content-consumer/providers/DataStatusProvider.provider.jsx